music/demos/callback_player.html

```diff
@@ -0,0 +1,83 @@
+<html>
```

Comment:
you need to add callback_player.ts to build_demos.sh and a link to this page from the index.html

Reply:
done